versions.properties

```diff
@@ -82,18 +60,11 @@ version.org.ops4j.pax.url..pax-url-aether=2.6.1
 version.org.spockframework..spock-core=1.0-groovy-2.4
 
 version.org.webjars..jquery=3.5.1
-##              # available=3.6.0
```

Comment:
Here you have to choose between upgrade to 3.6.0 or keep the new version reminder.